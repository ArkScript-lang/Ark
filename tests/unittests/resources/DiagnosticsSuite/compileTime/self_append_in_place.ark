(mut d [4 5 6])
(append! d d)
