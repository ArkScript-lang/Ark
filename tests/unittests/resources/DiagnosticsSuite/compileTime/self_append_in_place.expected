At d @ 2:12
    1 | (mut d [4 5 6])
    2 | (append! d d)
      |            ^
    3 |
        MutabilityError: Can not append! the list `d' to itself
